pulsar-grpc/src/main/proto/PulsarGrpc.proto

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+syntax = "proto3";
+
+package pulsar.grpc.proto;
+
+option java_multiple_files = true;
+option java_package = "org.apache.pulsar.grpc.proto";
+option java_outer_classname = "GrpcProxy";
+
+service Pulsar {
+    rpc Produce(stream ProducerMessage) returns (stream ProducerAck);
```

Comment:
does this have to be bidi streaming?

Reply:
doesn't simple request/response mode works better for produce?

Reply:
I mimicked the websocket behavior which does it this way. But it can be separated of course.

Reply:
> I mimicked the websocket behavior which does it this way. But it can be separated of course.

On second thought, I don't think it's possible to separate the streams as I think they need to use the same producer